x/auth/client/cli/tx_multisign.go

```diff
@@ -141,6 +150,7 @@ func makeMultiSignCmd() func(cmd *cobra.Command, args []string) (err error) {
 
 		sigOnly, _ := cmd.Flags().GetBool(flagSigOnly)
 
+		// TODO how i deal with aminoJSON here?
 		aminoJSON, _ := cmd.Flags().GetBool(flagAmino)
```

Comment:
Do i need to deal with aminoJSON here?

Reply:
Could you explain what you mean by "deal with aminoJSON"? For sure you shouldn't remove amino here.